build.gradle

```diff
@@ -201,6 +201,9 @@ dependencies {
 	// TOOLS.jar from jdk for javadoc support
     // compile files("$jdkHome/lib/tools.jar")
     compile files(System.getenv("JAVA_HOME") + "/lib/tools.jar")
+	dependencies {
```

Comment:
This doesn't look right.  There shouldn't be a nested dependency block...

Also, what is the reason for changing the path?

Reply:
On linux default JAVA_HOME env path is not set when you install openjdk 8, sorry for the nested dependency, fixing it